Src/GUI/Widgets/OutputFilesWidget.py

```diff
@@ -56,17 +78,17 @@ def get_widget_value(self):
                                     
         return (filename, formats)
     
-    def set_data(self, datakey):
-
-        basename = "output"
+    def set_data(self, datakey, analysis):
                 
         if datakey is None:

```

Comment:
I believe that `datakey` is only absent when a trajectory conversion is performed. Therefore, this option should never occur with this widget.

Reply:
About both configurators having names that are too close, what about renaming the `OutputFile` configure to `SingleOutputFile` ?

Reply:
You right. As the converter now use the `OutputFileWidget` instead of the `OutputFilesWidget`, we can safely remove the the `datakey  None` case